tools/instance_summary.rb

```diff
@@ -287,5 +287,73 @@ def get_summary
       print_grand_summary(grand_summary)
     end
   end
+
+  class PacketSummary < InstanceSummary
+    attr_accessor :packet
+
+    def initialize(jenkins: nil)
+      @packet = Packet.new
+      @jenkins = jenkins
+    end
+
+    # instances is <Array> of server objects
+    def summarize_instances(instances)
+      summary = []
+      instances.each do | inst |
+        inst_summary = {}
+        inst_summary[:region] = 'packet'
+        inst_summary[:name]= inst['hostname']
+        inst_summary[:uptime]= packet.instance_uptime inst["created_at"]
+        inst_summary[:owned] = inst['created_by']['email']
+        inst_summary[:flexy_job_id] = jenkins.get_jenkins_flexy_job_id(inst['hostname'])
+        summary << inst_summary
+      end
+      return summary
+    end
+
+    def get_summary
+      grand_summary = []
+      inst_details = packet.get_running_instances
+      summary = summarize_instances(inst_details)
+      # summary = summarize_instances(rg_name, instances)
+      print_summary(summary) if summary.count > 0
+      grand_summary << {platform: 'openstack', region: summary.first[:region], inst_count: summary.count}
+      print_grand_summary(grand_summary)
+    end
+  end
+
+
+  class VSphereSummary < InstanceSummary
+    attr_accessor :vms
```

Comment:
what does `vms` stand for here? I would expect to be `vsphere` or something. Or did I mistake the abbreviation?

Reply:
@akostadinov  vms==vSphere AFAIK, @jinyunma do you know why it's called vms?

Reply:
I will be surprised if in vmware community vms means vsphere but in either case that is confusing for me and probably other code contributers because it associates with virtual machines in generic context.

While `vsphere` will be much more legible. SO I suggest we switch regardless of vmware community conventions.